ecs_pod_creator.go

```diff
@@ -612,7 +617,7 @@ func (c *StoredRepositoryCredentials) SetPassword(pwd string) *StoredRepositoryC
 	return c
 }
 
-// Validate checks that both the username and password are set.
+// Validate checks that the secret name, username, and password are set.
```

Comment:
This is still just validating that the username and password are set, not the secret name.